definitions/marts/bigquery_marts/ecf/ecf_inductions.sqlx

```diff
@@ -169,7 +169,7 @@ FROM
     ecf_induction_records.updated_at,
     completion_date,
     induction_programme_details.cohort_id,
-    user_id,
+    ecf_participant_profiles.user_id,

```

Comment:
Why has this changed @consult1815 ? Not an issue, just interested to know why

Reply:
user_id also is available in the eligibility table, so just clarifying it should come from participant profiles

Reply:
Slay